server/odc-service/src/main/java/com/oceanbase/odc/service/integration/model/SSOIntegrationConfig.java

```diff
@@ -84,6 +88,13 @@ public static SSOIntegrationConfig of(IntegrationConfig integrationConfig, Long
                 parameter.fillParameter();
                 parameter.setSecret(integrationConfig.getEncryption().getSecret());
                 break;
+            case "LDAP":
+                Preconditions.checkArgument(integrationConfig.getEncryption().getEnabled()
```

Comment:
Why the encryption.enable should be `true` but the algorithm is `RAW`? RAW means no encryption.

Reply:
RAW represents that encryption is not required when making requests to external systems, but the local metadb still needs to store data in encrypted form.

Reply:
So why use one `Encryption` object to represent if excrypting data in two different scenario. It is pretty confusing.

Reply:
Solved.